api/config/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "port": 8080,
+  "dbUri": "postgres://postgres@127.0.0.1:5432/autoscaler",
```

Comment:
It would look and feel better if it was 
db {
  uri: xxx,
 maxConnections:
  minConnections:
  idleTimeout:
}